src/patterns/dosdp-patterns/nonhuman_disease_series_by_gene.yaml

```diff
@@ -0,0 +1,68 @@
+pattern_name: nonhuman_disease_series_by_gene
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/nonhuman_disease_series_by_gene.yaml
+
+description: 'This pattern is for non-human diseases that are defined based on a variation in one gene, such as many genetic diseases from the Online Mendelian Inheritance in Animals (OMIA).  Examples: TBD '
+
+contributors:
+- https://orcid.org/0000-0002-4142-7153
+- https://orcid.org/0000-0002-5002-8648
+
+classes:
+  disease: MONDO:0005583
+  gene: SO:0001217
+  taxon: NCBITaxon:1
+
+relations:
+  has material basis in germline mutation in: RO:0004003
+  in taxon: RO:0002162
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+
+vars:
+  disease: '''disease'''
+  gene: '''gene'''
+  taxon: "'taxon'"
+
+name:
+  text: '%s caused by variation in %s, %s'
+  vars:
+  - disease
+  - gene
+  - taxon
+
+annotations:
+- annotationProperty: exact_synonym
+  text: '%s %s, %s'
+  vars:
+  - gene
+  - disease
+  - taxon
+- annotationProperty: exact_synonym
+  text: '%s related %s, %s'
+  vars:
+  - gene
+  - disease
+  - taxon
+- annotationProperty: exact_synonym
+  text: '%s caused by mutation in %s, %s'
+  vars:
+  - disease
+  - gene
+  - taxon
+
+def:
+  text: Any %s, occuring in %s, in which the cause of the disease is a variation in the %s gene.
+  vars:
+  - disease
+  - taxon
+  - gene
+
+equivalentTo:
+  text: '%s and (''has material basis in germline mutation in'' some %s) and (''in taxon'' some %s)'
```

Comment:
Note here that there are two ways you can model this:

1. you can push the taxon into the gene. assuming you always use hgnc identifiers for humans and ncbigenes for animals, and you always use the correct gene id pertaining to a taxon specific gene, this is enough, no need for recording the taxon specifically to distinguish from human disease
2. however, you likely want the taxon to separate grouping classes "dog disease", which means having the taxon can be helpul (unless the genes in our ingest have them already!). You might also need this for non-gene non-human-animal diseases, so maybe just curate redundantly. If you do, however leave it like you have it, not that _technically_, you may want to add the human taxon to the human diseases. 

Long story short: its probably best to leave it has you have it, just adding some considerations in for future decision making.

Reply:
After a lot of discussion, we decided to keep the redundancy. But I agree, we might decide to remove this redundancy at a later time